### ISSUE TITLE
Maximum packet size is 65535 bytes? How will I stream images?

### ISSUE THREAD

**Issue by shrikky** [j0r1/JRTPLIB#55]
The max packet size seems to be 65535, I am not able to change it even with setMaxPacketSize function

**Comment by Kartofi** [comment]
So almost 2 years later I am here to answer your question,
So the maximum size of a packet is 65535 bytes which is around 65 kb +- so if you can fit an image in it you are a magician.There is not magic so the only choice we have is to just split the image into parts that are 65 kb and send them then we just merge them boom

**Comment by shrikky** [comment]
> 

Yes i did divide the image into chunks and reconstructed on the client side.

**Comment by Kartofi** [comment]
Good job and what was the project about
